app/controllers/report_controller.rb

```diff
@@ -354,6 +288,71 @@ def cancel_import
 
   private ###########################
 
+  def set_active_elements(feature)
+    if feature
+      self.x_active_tree ||= feature.tree_list_name
+      self.x_active_accord ||= feature.accord_name
+    end
+    get_node_info
+  end
+
+  def features
+    [{:role        => "miq_report_saved_reports",
+      :role_any    => true,
+      :name        => :savedreports,
+      :accord_name => "savedreports",
+      :tree_name   => :savedreports_tree,
+      :title       => N_("Saved Reports"),
+      :container   => "savedreports_accord"},
+
+     {:role        => "miq_report_reports",
+      :role_any    => true,
+      :name        => :reports,
+      :accord_name => "reports",
+      :tree_name   => :reports_tree,
+      :title       => N_("Reports"),
+      :container   => "reports_accord"},
```

Comment:
I also wonder why we have here `_accord` while the default is `_tree_div`. Is there a reason for the difference? Can we unify this?